snuba/datasets/storages/sessions.py

```diff
@@ -115,21 +116,23 @@ def process_query(self, query: Query, request_settings: RequestSettings) -> None
             )
 
 
+partially_built_stream_loader = partial(
+    build_kafka_stream_loader_from_settings,
+    processor=SessionsProcessor(),
+    default_topic=Topic.SESSIONS,
+    dead_letter_queue_topic=Topic.DEAD_LETTER_SESSIONS,
+)
```

Comment:
Can you just put this inline below (even if there's a little duplication)? Sessions subscriptions is about to be killed and it'll make it easier to clean up.